bin/govuk-docker

```diff
@@ -1,23 +1,44 @@
-#!/usr/bin/env ruby
+#!/bin/bash
 
-require "pathname"
-ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
-                                           Pathname.new(__FILE__).realpath)
+case $1 in
+  doctor)
+    echo '*** DEPRECATED: use bundle exec bin/doctor instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/docs/installation.md#how-to-troubleshoot-your-installation'
+    exit 1
+    ;;
 
-bundle_binstub = File.expand_path("govuk_docker_bundle", __dir__)
+  setup)
+    echo '*** DEPRECATED: use bundle exec bin/setup instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/docs/installation.md#setup'
+    exit 1
+    ;;
 
-if File.file?(bundle_binstub)
-  if File.read(bundle_binstub, 300) =~ /This file was generated by Bundler/
-    load(bundle_binstub)
-  else
-    abort("Your `bin/bundle` was not generated by Bundler, so this binstub cannot run.
-Replace `bin/bundle` by running `bundle binstubs bundler --force`, then run this command again.")
-  end
-end
+  build)
+    echo '*** DEPRECATED: use make <service> instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/README.md#usage'
+    exit 1
+    ;;
+esac
 
-require "rubygems"
-require "bundler/setup"
+echo '*** DEPRECATED: use exe/govuk-docker instead ***'
+echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/docs/installation.md#setup'
+echo
 
-require_relative "../lib/govuk_docker"
-require_relative "../lib/govuk_docker/cli"
-GovukDocker::CLI.start(ARGV)
+case $1 in
+  startup)
+    echo '*** DEPRECATED: use exe/govuk-docker up <service>-<stack> instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/docs/installation.md\#shortcuts'
+    ;;
+
+  run)
+    echo '*** DEPRECATED: use exe/govuk-docker run <service>-<stack> instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/docs/installation.md\#shortcuts'
+    ;;
+
+  prune)
+    echo '*** DEPRECATED: use exe/govuk-docker rm -sv instead ***'
+    echo 'https://github.com/alphagov/govuk-docker/blob/e00ab054d8d94132f1539b0027eefbe44e168f6f/README.md#how-to-clear-your-docker-containers'
+    ;;
+esac
+
+exit 1
```

Comment:
This looks really helpful for communicating the change 👍 

Couple of things: 
- we shouldn't use the terminology DEPRECATED here as this goes further than deprecation and has removed the functionality from these commands. Maybe replace with DEPRECATED with MOVED"?
- might be better to link to master branch rather than the commit SHA to reflect most up-to-date version of documentation.

Reply:
I was thinking the SHA would be more stable, as the stuff on master might change if this gets merged in, and it might be more helpful to link to something that was valid at a point in time. In some cases, like installation, the master version doesn't exist yet, so this seems like the way to go for the time being.

Reply:
> I was thinking the SHA would be more stable, as the stuff on master might change if this gets merged in, and it might be more helpful to link to something that was valid at a point in time. In some cases, like installation, the master version doesn't exist yet, so this seems like the way to go for the time being.

Yup, both approaches have downsides. The SHA hash seems fine if this doesn't hang around too long otherwise it runs the risk of pointed to superseded info.